core/kits/manager.php

```diff
@@ -151,6 +151,54 @@ public function get_kit_for_frontend() {
 		return $kit;
 	}
 
+	/**
+	 * Map Scheme To Global
+	 *
+	 * Convert a given scheme value to its corresponding default global value
+	 *
+	 * @param string $type 'color'/'typography'
+	 * @param $value
+	 * @return mixed
+	 */
+	private function map_scheme_to_global( $type, $value ) {
```

Comment:
maybe isset here?

Reply:
Actually there is no reason that the old scheme will point to a value that doesn't exist